drs/drs-assembly/subscribe-model.conf

```diff
@@ -313,17 +369,44 @@ IRIS, TCS, ESW, NFIRASO, AOESW and NSCU to be associated with science data
     {
       subsystem = "IRIS"
       component = "sc.cryoenv"
-      name = "IMGTEMP[XX]"
+      name = "imgDetectorTemperature"
```

Comment:
I did a test ingest of these changes and took a look with icdwebserver. I noticed that this block of events that you subscribe to from the sc.cryoenv assembly appear not to be published. Just curious if these are proposed changes and you will ask the owner of that assembly to produce the requested items?

Reply:
@jimansohn can chime in, but those items are defined in his working branch:
https://github.com/tmt-icd/IRIS-Model-Files/blob/sohn_cryoUpdate/sc/cryoenv-assembly/publish-model.conf

Reply:
Yes these changes reflect the updates on Ji Man's branch recently.  I think the current pull request looks fine.

Reply:
I created PR #37 to avoid error when master is ingested.